manifests/m/Microsoft/OpenJDK/17/17.35.1/Microsoft.OpenJDK.17.locale.en-US.yaml

```
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.4.1.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+PackageIdentifier: Microsoft.OpenJDK.17
+PackageVersion: 17.35.1
+PackageLocale: en-US
+Publisher: Microsoft
+PublisherUrl: https://www.microsoft.com
+PublisherSupportUrl: https://support.microsoft.com
+PrivacyUrl: https://privacy.microsoft.com/en-us/privacystatement
+Author: Microsoft
+PackageName: Microsoft Build of OpenJDK with Hotspot 17+35 (x64)
+PackageUrl: https://www.microsoft.com/openjdk
+License: GPL2 with Classpath Exception
+LicenseUrl: https://openjdk.java.net/legal/gplv2+ce.html
+ShortDescription: The Microsoft Build of OpenJDK is a new no-cost long-term supported distribution and Microsoft"s new way to collaborate and contribute to the Java ecosystem.
```

```suggestion
ShortDescription: The Microsoft Build of OpenJDK is a new no-cost long-term supported distribution and Microsoft's new way to collaborate and contribute to the Java ecosystem.
```

I'm 80% sure this is my fault, but it still needs to be fixed :-)

---

No blame game here, friend! 😅

---

(Couldn't repair the older manifests in this PR 🤦‍♂️)